### DISCUSSION TITLE
how do I change the default directory for the *.cpp:tests files that are generated automatically

### DISCUSSION THREAD

**Discussion by Siddhant-sarkar** [Jatana/FastOlympicCoding#65] [Q&A]
I just want to keep my files clean and for that, I just want to keep the *.cpp: tests files in a different directory. Please help.

**Comment by npanuhin** [comment]
`tests_relative_dir` and `tests_file_suffix` settings are there for you. You might want to ~install the package directly from https://github.com/Jatana/FastOlympicCoding by adding it as a repository in Package Control~ upgrade the package as usual (<kbd>Ctrl+Shift+P</kbd> -> Upgrade Package)

@Jatana, can you please create a new tag, to update Package Control?

**Comment by Jatana** [comment]
The new tag [v1.4.12](https://github.com/Jatana/FastOlympicCoding/releases/tag/v1.4.12) has been released.
